pyEpiabm/pyEpiabm/intervention/abstract_intervention.py

```diff
@@ -0,0 +1,45 @@
+#
+# AbstractIntervention Class
+#
+
+
+class AbstractIntervention:
+    """Abstract class for Interventions.
+
+    """
+    def __init__(self, start_time, policy_duration, threshold, population):
+        """Set the parameters of the interventions
+
+        Parameters
+        ----------
+        start_time : float
+            Start time of intervention
+        policy_duration : float
+            Duration of the intervention
+        threshold : float
+            Number of cases required to trigger the intervention
+        is_active : boolean
+            Whether intervention is active
+        """
+        self.start_time = start_time
+        self.policy_duration = policy_duration
+        self.threshold = threshold
+        self._population = population
+
+    def is_active(self, time, num_cases):
```

Comment:
Even short methods like this should have full documentation (in the style you have above) with a short description of what the function does, the input parameters and the return value. If you're unsure how to structure this, I would recommend checking out previous examples in the code.